src/style.css

```diff
@@ -0,0 +1,130 @@
+body {
+  margin: 0;
+  background-color: #e5e5e5;
+  font-family: "Roboto", sans-serif;
+
+  --white-color: #fff;
+  --blue-color: #00acdc;
+  --light-grey-color: #616070;
+  --dark-grey-color: #060b35;
+}
+
+.card {
+  box-sizing: border-box;
+  width: 200px;
+  background-color: var(--white-color);
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  display: flex;
+  flex-direction: column;
+  padding: 32px 16px 16px;
+}
+
+.card__image {
+  width: 160px;
+  height: 134px;
+  align-self: center;
+}
+
+.card__title {
+  margin: 40px 0 0;
+  color: var(--dark-grey-color);
+  font-size: 12px;
+  font-weight: 500;
+  line-height: 18px;
+}
+
+.card__code {
+  font-weight: 400;
```

Comment:
font-weight: 400; is a default font-weight (if you haven't passed sth another to your page). So, no needs to use it for other selectors as well
```suggestion
```